src/infrastructure/api/fetch_Items.ts

```diff
@@ -1,12 +1,12 @@
 import axios from 'axios';
 
-const apiCallFunc = async() => {
+const fetchItems = async() => {
   const token: string = '1k5sbhafeq3stb477djbommo9z2vrwj40x1zymsa'
   const apiCall = await axios.get('https://api.json-generator.com/templates/NM0ZuloCu1TZ/data', {headers: {"Authorization" : `Bearer ${token}`}})
     .then(res => res.data)
-    .catch(err => console.error(err));
+    .catch(err => alert(`Unknown error occured! \n${err}`));
```

Comment:
Instead of alert - you should `throw 'error'`
Then, in the presentation layer, handle the error by showing an error page - not an alert